### ISSUE TITLE
Import/Overwrite “CleanTitle, The” in Radarr/Sonarr Media Management

### ISSUE THREAD

**Issue by Del-Amitri** [Casvt/Plex-scripts#75]
Currently in Radarr and Sonarr Media Management (I’ll focus on Radarr for simplicity), in the Movie Naming Sections “Standard Movie Format” and “Movie Folder Format”, you can structure the naming formats to have a “CleanTitle” or a “Title, The” format, but not both. CleanTitle removes non uniform characters, but it still misses a lot. “Title, The” puts the ‘The’ after the rest of the title of the movie. 

For instance, CleanTitle names ‘¡Three Amigos!’ ‘Three Amigos!’ but misses Amélie, whereas it doesn’t remove the é. So it’s like 60%-75% solid

“Movie, The” works as intended but cannot be combined with the aforementioned “CleanTitle”. So if a movie is called “The Höuse” it could only be imported as “Höuse, The” or “ The House”, but not “House, The”.

Per your Reddit comment, it would be more robust than just renaming a folder based off the file or visa versa because I, and I’m sure many others, use different naming structures for the movie file and the folder it sits in. For instance my structure for folders is “Title, The (year) (IMDb code). And my files are “ CleanTitle (Year) (Edition Tag) (Video Quality) (Video Codec) (Audio Codec) (Audip Channels) (Release Group) (IMDb ID)”. This is done so that the folder always stays the same after upgrades, and files have in-depth info in their title.

So basically it comes down to how Radarr and Sonarr create the folder and import the files, and if there’s a way to intercept it. Or at least fix it after it’s done. As always, no sweat if this isn’t possible! Thanks!

**Comment by Casvt** [comment]
Too niche. Just open a ticket at Sonarr and Radarr
